File: hw-1B/Python/A-interactor.py
def output(r, i, c):

    if i == 0:
        if r != 0:
            return 3
        return c
    elif i == 1:
        return c
    elif i == 4:
        if r != 0:
            return 3
        return 4
    elif i == 6:
        return 0
    elif i == 7:
        return 1
    return i


if __name__ == '__main__':

    "Output the result of the testing system"

    r = int(input())
    i = int(input())
    c = int(input())
    res = output(r, i, c)
    print(res)

File: hw-1/C++/A-interactor.cpp
# include <iostream>

using namespace std;

int output(int r, int i, int c){

    if (i == 0){
        if (r != 0)
            return 3;
        return c;
    }
    else if (i == 1){
        return c;
    }
    else if (i == 4){
        if (r != 0)
            return 3;
        return 4;
    }
    else if (i == 6){
        return 0;
    }
    else if (i == 7){
        return 1;
    }
    return i;
}

int main()
{
    //Output the result of the testing system

    int r; int i; int c;
    cin >> r;
    cin >> i;
    cin >> c;
    int res = output(r, i, c);
    cout << res << endl;
    return 0;
}
